### PR TITLE
Fetch & Display Message from Server

### PR DESCRIPTION
**Pull request by andrewhojel**
milestone requested we submit a PR 

important changes:
- use fetch to get the text data from the url/data doGet function
- slice the message to remove formatting (so that if you go to url/data the formatting remains)
- display the cleaned data from server on index.html

### CODE REVIEW

**Review by Caffeinix — COMMENTED** [review]
It looks like this change is based on the previous version of your code...  you might need to git pull so git knows that some of these changes were already approved from your previous PR.

**Comment by andrewhojel** [comment]
@Caffeinix do you mean I should pull when I switch to a new branch? Since the only pushes are coming from me shouldn't my code always be the most up to date?

**Comment by Caffeinix** [comment]
I'm not quite sure how this maps to the process on your side, but what I'm seeing is that changes from a previous PR are showing up as part of the diff in this one as well.  Pulling won't switch you to a new branch, it'll just merge the changes that have been merged into that upstream branch into your local version.

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -44,6 +44,16 @@ body {
...
+/* style h2 */
+h2 {
+    font-weight: normal;
+}
+
+/* style h4 */
+h4 {
+    font-weight: normal;
+}
+
```
Comment (jessimb) [inline portfolio/src/main/webapp/style.css:56]: Same as the other PR, if you fix it there and merge it, I expect pulling will fix it here too.

**Review by jessimb — APPROVED** [review]
I don't want this to block you as we navigate the github oddities, so I'll approve it.

**Pull request merged**